changelog/12497.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed two failing pdb-related tests on Python 3.13.
```

Comment:
Given this did not fix anything on pytest itself, only in the test suite, perhaps making this a `misc` changelog entry would be more appropriate?

Reply:
We have `contrib` too, which is described as:

> stuff that affects the contributor experience. e.g.
> Running tests, building the docs, setting up the development
>  environment.

That seems like a good fit given the "running tests" example.

Reply:
Agreed! 👍